### PR TITLE
Implemented optional clickable tr

### PR DESCRIPTION
**Pull request by meijdenmedia**
When the tr is fully clickable it is hard to copy some of the exception / context. I added an optional .env variable which can disable the fully clickable functionality.

### CODE REVIEW

**Comment by meijdenmedia** [comment]
Break on all characters accidently also came in with this PR, will remove that tomorrow.

**Comment by rap2hpoutre** [comment]
Thanks for your contribution and sorry for late answer! I feel like it does not need to be an option but a decision _by design_ in the library. And I'm OK with you: with the whole line clickable, it's hard to select text. But, some time ago, I accepted a PR for this: https://github.com/rap2hpoutre/laravel-log-viewer/pull/91. I accepted it because some people up-voted it (with emoji thumb up), so I thought: "Ok, people prefer this".

So basically: I don't want to add an option for this, but make a decision; I prefer the whole line not to be clickable, but I'm not sure I will change it...

I close this for now, feel free to ask for reopen.
